lib/db/schema/patch-026-027.sql

```diff
@@ -0,0 +1,43 @@
+DROP PROCEDURE `openIdRecord_1`;
+DROP PROCEDURE `createAccount_3`;
```

Comment:
I support "going backwards with versions seems wrong", let's make it `createAccount_4`.